docs/rest_api.rst

```diff
@@ -367,7 +369,7 @@ Channel Management
     - The provided JSON is in some way malformed, or
     - there is nothing to do since neither ``state`` nor ``total_deposit`` have been given, or
     - the value of ``state`` is not a valid channel state.
-   :statuscode 402: Insufficient balance to do a deposit
+   :statuscode 402: Insufficient balance to do a deposit, or insufficient ETH to pay for gas for the on-chain transaction
```

Comment:
I'd rather say `insufficient ETH to pay for gas of the on-chain transaction`

Reply:
Agreed. Done!